### PR TITLE
BXMSPROD-1038: Create short release pipeline just for drools

### PR DESCRIPTION
**Pull request by mbiarnes**
**Thank you for submitting this pull request**

**JIRA**: _(please edit the JIRA link if it exists)_ 

[BXMSPROD-1038](https://issues.redhat.com/browse/BXMSPROD-1038?filter=-1)

**referenced Pull Requests**: _(please edit the URLs of referenced pullrequests if they exist)_

* referenced pull request 1
* referenced pull request 2
* referenced pull request 2
etc. 

<pre>
How to retest a PR or trigger a specific build:

* <b>a pull request</b> please add comment: <b>Jenkins retest this</b>
</pre>

### CODE REVIEW

File: job-dsls/jobs/reduced_drools_release.groovy [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+                    sh  'pwd \\n' +
+                        'git branch \\n' +
+                        'git checkout -b $baseBranch'
+                }
+            }    
+        }
+        stage ('Clone others') {
+            steps {
+                sshagent(['kie-ci-user-key']) {
+                    sh 'sh droolsjbpm-build-bootstrap/script/release/01_cloneBranches.sh $baseBranch'
```
Comment (mareknovotny) [inline job-dsls/jobs/reduced_drools_release.groovy:48]: so we are cloning all repository list? Where is the preparation of reduced repository list?

File: job-dsls/jobs/reduced_drools_release.groovy [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+                }
+            }
+        }
+        // part of the Maven rep will be erased
+        stage ('Remove M2') {
+            steps {
+                sh "sh droolsjbpm-build-bootstrap/script/release/eraseM2.sh $m2Dir"
+            }
+        }
+        stage ('replace repository-list.txt and change branched-7-repository-list.txt') {
```
Comment (mareknovotny) [inline job-dsls/jobs/reduced_drools_release.groovy:93]: Shouldn't this stage be more up, before you clone the other repositories?

File: job-dsls/jobs/reduced_drools_release.groovy [hunk 3]
```diff
@@ -0,0 +1,241 @@
...
+                           'cat droolsjbpm-build-bootstrap/script/repository-list.txt \\n' +
+                           'rm droolsjbpm-build-bootstrap/script/branched-7-repository-list.txt \\n' +
+                           'touch droolsjbpm-build-bootstrap/script/branched-7-repository-list.txt \\n' +
+                           'echo "optaplanner" >> droolsjbpm-build-bootstrap/script/branched-7-repository-list.txt \\n' +
+                           'cat droolsjbpm-build-bootstrap/script/branched-7-repository-list.txt'
+                    }       
+                }
+            }
+        }         
+        stage ('meet github'){
```
Comment (mareknovotny) [inline job-dsls/jobs/reduced_drools_release.groovy:107]: the stage name is little bit meaningless ;)
What about Configure github user?

File: job-dsls/jobs/reduced_drools_release.groovy (hunk 2 above)
Comment (mbiarnes) [reply job-dsls/jobs/reduced_drools_release.groovy:93]: right now it is in a later step - but you're right, will change this

File: job-dsls/jobs/reduced_drools_release.groovy (hunk 3 above)
Comment (mbiarnes) [reply job-dsls/jobs/reduced_drools_release.groovy:107]: OK

File: job-dsls/jobs/reduced_drools_release.groovy (hunk 1 above)
Comment (mbiarnes) [reply job-dsls/jobs/reduced_drools_release.groovy:62]: fixd: look at stage replace repository-list.txt and change branched-7-repository-list.txt

File: job-dsls/jobs/reduced_drools_release.groovy [hunk 4]
```diff
@@ -0,0 +1,241 @@
...
+                        'git branch \\n' +
+                        'git checkout -b $baseBranch'
+                }
+            }    
+        }
+        stage ('Replace repository-list.txt and change branched-7-repository-list.txt') {
+            steps {
+                configFileProvider([configFile(fileId: '1a43573a-318c-426a-bb2b-c9df7fe97a02', targetLocation: 'repository-list.txt', variable: 'REP_LIST')]) {
+                    dir("${WORKSPACE}") {
+                        sh 'cp repository-list.txt droolsjbpm-build-bootstrap/script/repository-list.txt \\n' +
```
Comment (mareknovotny) [inline job-dsls/jobs/reduced_drools_release.groovy:49]: Where does this repository-list.txt file come from?

Comment (mareknovotny) [reply job-dsls/jobs/reduced_drools_release.groovy:49]: so answering my own question, there is managed file on Jenkins with reduced content

**Review by mareknovotny — APPROVED** [review]

**Pull request merged**